app.rb

```diff
@@ -31,10 +42,12 @@ def create_student
     print 'Name: '
     name = gets.chomp
     print 'Has parent permission? [Y/N]: '
-    parent_permission = gets.chomp
-    person = Student.new(name, age, parent_permission)
+    parent_permission = gets.chomp.downcase == 'y'
+    id = Random.rand(1...1000)
+    person = Student.new(name, age, parent_permission: parent_permission, id: id)
     @people.push(person)
     puts "Student '#{name}' created successfully"
+    save_people_to_json

```

Comment:
- [ ] [OPTIONAL]
The _people.json file_ is reset when the app loads, let's ensure that if there are already saved data in this file it should be reloaded when the app starts.